1. Desktop app calls Web API/TodoListClient/MainWindow.xaml.cs

```diff
@@ -287,12 +285,19 @@ private async void SignIn(object sender = null, RoutedEventArgs args = null)
             {
                     // Force a sign-in (Prompt.SelectAccount), as the MSAL web browser might contain cookies for the current user
                     // and we don't necessarily want to re-sign-in the same user
-                    var result = await _app.AcquireTokenInteractive(Scopes)
+                    var builder = _app.AcquireTokenInteractive(Scopes)
                         .WithAccount(accounts.FirstOrDefault())
-                        .WithPrompt(Prompt.SelectAccount)
-                        .WithUseEmbeddedWebView(true)
-                        .ExecuteAsync()
-                        .ConfigureAwait(false);
+                        .WithPrompt(Prompt.SelectAccount);
+
+                    if (!_app.IsEmbeddedWebViewAvailable())
```

Comment:
If this is too complicated for this tutorial, propose to revert to using .NET classic

Reply:
I think it's fine;